src/common/modules/data/DefaultSettings.js

```diff
@@ -19,13 +19,16 @@ const defaultSettings = {
         enabled: false,
         autocorrectSymbols: true,
         autocorrectUnicodeQuotes: true,
-        autocorrectUnicodeFracts: true,
+        autocorrectUnicodeFracts: true
     },
     unicodeFont: {
         changeFont: true,
         changeCase: true,
         showReadableText: false,
         livePreview: false
+    },
+    notifications: {
```

Comment:
BTW is there a good reason to have this in a nested object?

Otherwise `sendNotifications` or better naming possibly `sendInstallAndUpdateNotifications` or `sendInstallAndUpgradeNotifications` may be better here.

Reply:
I thought we might want to add more fine grained options to the Notification module in the future. This option currently controls all notifications, including both these "install and update" and the "Thunderbird specific workaround for not being able to access the subject and other fields in the compose window" notifications added in #72, as well as any future tip notifications added as part of https://github.com/TinyWebEx/RandomTips/issues/9. Anyway, I would be fine either way.

Reply:
Note you can use `InstallUpgrade.js` to later transform these options. Anyway, this is okay of course.